### PR TITLE
add .py to test pr 502 bmc and smbios info

### PR DESCRIPTION
**Pull request by willwu1**

### CODE REVIEW

**Comment by willwu1** [comment]
@weia5-emc @XiaowenJiang  @chenge3 plz help to review, thanks

File: test/functional/test_fru_info.py [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+def teardown_module():
+    if conf:
+        stop_node()
+    os.environ["PATH"] = OLD_PATH
+
+
+def start_node():
+    """
+    create two drive for comparasion.
```
Comment (weia5-emc) [inline test/functional/test_fru_info.py]: Could you please remove these 2 lines.

**Comment by XiaowenJiang** [comment]
+1

File: test/functional/test_fru_info.py (hunk 1 above)
Comment (willwu1) [reply test/functional/test_fru_info.py]: ok,thanks

**Comment by weia5-emc** [comment]
good job

**Pull request merged**